src/mappening/api/utils/tokenize.py

```diff
@@ -1,52 +1,41 @@
 from flask import Flask, jsonify, request, json, Blueprint
-from flask_cors import CORS, cross_origin
 
 import nltk
 nltk.download('perluniprops')
 nltk.download('nonbreaking_prefixes')
 nltk.download('stopwords')
 nltk.download('punkt')
 
-import re
-import string
-from tqdm import tqdm
 from nltk.tokenize.moses import MosesTokenizer, MosesDetokenizer
-
 from nltk.corpus import wordnet as wn
+from tqdm import tqdm
+import string
+import re
 
-Process = Blueprint('Process', __name__)
-
-# Enable Cross Origin Resource Sharing (CORS)
-cors = CORS(Process)
+tokenize = Blueprint('tokenize', __name__)
 
-@Process.route('/api/process/<text>', methods=['GET'])
-def processText(text):
+@tokenize.route('/<text>', methods=['GET'])
```

Comment:
Why is this a route

Reply:
Leftovers from testing location things. It was convenience, but I don't need it anymore.